internal/attributed-string-slice/attributed_string_slice.go

```diff
@@ -0,0 +1,79 @@
+package main
+
+import (
+	"bytes"
+	"fmt"
+	"reflect"
+
+	"github.com/BurntSushi/toml"
+)
+
+type attributedStringSlice struct { // A "mixed-type array" in TOML.
```

Comment:
Shouldn't this be a public type?

Reply:
Yes, we may need to make the type public at some point when integrating it into containers.conf.

Here, I just want to show how (and that) it can be done.  We can recycle/move the code when integrating it into contianers.conf.